### ISSUE TITLE
[Commande] Modération

### ISSUE THREAD

**Issue by rick-gnous** [TobiBiotex/CDS-BOT#2]
- [ ] kick <mention user | id>
- [ ] ban <mention user | id>
- [ ] unban <mention user | id>
- [ ] mute <mention user | id>
- [ ] unmute <mention user | id>
- [ ] purge/clean <nombre de messages> : supprimer les x derniers messages

**Comment by rick-gnous** [comment]
ticket modifié

**Comment by rick-gnous** [comment]
oui, github a la facheuse tendance à cacher les mots entre `<>` s’il n’y a pas d’espace.

pour mute/unmute, ce serait mute aussi bien en vocal qu’en textuel et un role permettra de simplifier les permissions (juste à faire une attribution, pas besoin de modifier tous les salons). @TobiBiotex on est OK sur ça ?

**Comment by Kekwel** [comment]
Ok d'acc, il faudra revoir alors les permissions à créer car sur le serveur CDS il y a genre que 3 rôles (Admin, Modo et Papa Noël ^^), sans compter le bot.

Car je viens de tester, si on donne un rôle "Muet" à qqun avec l'interdiction d'envoyer des messages, si le rôle @ everyone (permissions par défaut) autorise l'envoi de message, ca écrase l'interdiction du rôle "Muet".. ou alors je m'y prends mal avec la config Discord
